content/05kapitel.tex

```diff
@@ -2,8 +2,118 @@
 
 \chapter{Technische Umsetzung}
 
-\section{Mockups}
 
 \section{Architektur}
 
+\subsection{Übersicht der Services}
+Unser Produkt ist in mehrere Services aufgeteilt. Die Services sind \textit{Core-Service}, \textit{Polling-Service} und das \textit{Frontend}.
+Desweiteren gibt es eine \textit{PostgreSQL Datenbank} und eine \textit{Keycloak} Instanz. Die letzten beiden Services sind Abhänigkeiten von unseren entwickelten Services.
+Die Datenbank wird dafür verwendet um Daten wie RSS-Feeds und Artikel zu speichern.
+Keycloak wird dafür verwendet um den \textit{Core-Service} und das \textit{Frontend} abzusichern und die Benutzer zu verwalten.
+Die Hauptaufgabe des \textit{Core-Services} ist es RSS-Feeds und Artikel zurverfügung zu stellen, dies geschieht mittels \textit{GraphQL}. Desweiteren bietet er die Möglichkeit
+neue RSS-Feeds anzulegen, oder auch Abonnents, Favouriten und Lesezeichen eines Benutzers zu verwalten. Der \textit{Polling-Service} ist für die automatische Datenabfrage von RSS-Feeds und
+deren Artikel, diese werden anschließend in der Datenbank gespeichert. Das \textit{Frontend} ist die Endschnittstelle unseres Produkts zu dem Benutzer.
```

Comment:
deren Artikel zuständig. Diese werden anschließend in der Datenbank gespeichert

Reply:
sollte gefixed sein